flake8_trio.py

```diff
@@ -90,11 +144,14 @@ def visit_nodes(
                 for node in arg:
                     visit(node)
 
-    def error(self, error: str, node: HasLineInfo, *args: Any, **kwargs: Any):
+    def error(
+        self,
+        error: str,
+        node: HasLineInfo,
+        *args: Union[HasLineInfo, TrioScope, str, int],
```

Comment:
I'd usually just annotate that `Any`, or `object`, because it looks like a "whatever we get passed" argument and ultimately gets formatted into a string.

Reply:
I had some reason for wanting to typecheck this when I wrote this, though it might've gotten lost in a refactor of `test_flake8_trio` and yeah it's pretty messy anyway.
If anything some kind of "is_stringable"